src/transformers/modeling_prophetnet.py

```diff
@@ -288,11 +289,18 @@ class ProphetNetSeq2SeqLMOutput(ModelOutput):
     decoder_ngram_hidden_states: Optional[Tuple[torch.FloatTensor]] = None
     decoder_attentions: Optional[Tuple[torch.FloatTensor]] = None
     decoder_ngram_attentions: Optional[Tuple[torch.FloatTensor]] = None
-    decoder_cross_attentions: Optional[Tuple[torch.FloatTensor]] = None
+    cross_attentions: Optional[Tuple[torch.FloatTensor]] = None
     encoder_last_hidden_state: Optional[torch.FloatTensor] = None
     encoder_hidden_states: Optional[Tuple[torch.FloatTensor]] = None
     encoder_attentions: Optional[Tuple[torch.FloatTensor]] = None
 
+    @property
+    def decoder_cross_attentions(self):
+        warnings.warn(
+            "`decoder_cross_attentions` is deprecated and will be removed soon. Please use `cross_attentions` instead."
+        )
```

Comment:
Make it a `FutureWarning` so that it's shown.

(Not sure if really need it anyway because it's more of an internal class, I doubt users are instantiating `ModelOutput` themselves.)